q2_phylogeny/tests/test_raxml.py

```diff
@@ -24,6 +26,25 @@ class RaxmlTests(TestPluginBase):
 
     package = 'q2_phylogeny.tests'
 
+    @classmethod
+    def setUpClass(cls):
+        """Copies test data to `data_dir` and defines it as an attribute on
+        `RaxmlTests`."""
+        tmpdir = tempfile.mkdtemp()
```

Comment:
I think you're missing a call to `super()` here - we don't need to override the parent class' `setUpClass`, we are just supplementing it with more behavior. This doesn't matter at the moment, but will matter in the future, if and when `TestPluginBase` gets its own customization in `setUpClass`, so I think adding now will be helpful for our future selves.

Reply:
Good point 👍

Reply:
Is this how `super` should be called, @thermokarst?

Reply:
LGTM! There is also a shorthand syntax that was introduced more recently that doesn't require the explicit declaration of the parent class, but its neither here nor there.